### PR TITLE
docs: add step to create symlink

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**

Enhancement.

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
Yes

**Summary**

The motivation for this change -  [comment](https://github.com/webpack/webpack-cli/issues/1165#issuecomment-599445409).

Screenshot from 2020-03-28 08-23-47

**Does this PR introduce a breaking change?**

NO

**Other information**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -73,13 +73,23 @@ In case you are suggesting a new feature, we will match your idea with our curre
...
 
+-   Create a symlink of package. [Learn More](https://classic.yarnpkg.com/en/docs/cli/link/)
+
+    ```bash
+    yarn link
+
+    //now go to the location you want to test the package.
+
+    yarn link webpack-cli
+    ```
```
Comment (evilebottnawi) [inline .github/CONTRIBUTING.md]: I don't do it and all works fine, what is a problem?

Comment (snitin315) [reply .github/CONTRIBUTING.md]: Actually some new contributors get confused with how to test their changes. That's why added this step. 
[For example Refer this comment](https://github.com/webpack/webpack-cli/issues/1165#issuecomment-599445409)

Comment (snitin315) [reply .github/CONTRIBUTING.md]: Good to have so anyone can look in docs instead of asking someone.

Comment (anshumanv) [reply .github/CONTRIBUTING.md]: I think we're good as is.

**Pull request closed without merging**